OpenUtau/Strings/Strings.axaml

```diff
@@ -8,6 +8,8 @@
   <system:String x:Key="context.part.delete">Delete part</system:String>
   <system:String x:Key="context.part.rename">Rename part</system:String>
   <system:String x:Key="context.part.replaceaudio">Reselect audio file</system:String>
+  <system:String x:Key="context.part.transcribe">Transcribe</system:String>
```

Comment:
Even though `transcribe` is correct here, a more specific description like "Transcribe audio to create a note part" could be helpful.